web/impact/impact/settings.py

```diff
@@ -161,7 +161,8 @@ class Base(Configuration):
         'localhost:8181',
     )
     CORS_ORIGIN_REGEX_WHITELIST = (
-        r'^(https?://)?(\w+\.)?masschallenge\.org$', )
+        r'^(https?://)?(\w+\.)?masschallenge\.org$',
+        r'^(https?://)?(\w+)(-)?(\w+)?.masschallenge\.org$', )
```

Comment:
does the "." just before "masschallenge" need to be escaped?

Reply:
pre-staging works without escaping the `.`

so I'd say, no. We don't need to escape this

Reply:
while it will validate on the cases we care about without the escape this regex matches any domain so long as it ends in "masschallenge.org" (ex: https://maliciousdomainmasschallenge.org)

Reply:
ohh, I see what you mean! good catch! fixing now

Reply:
fixed.